quantus/metrics/robustness/relative_representation_stability.py

```diff
@@ -22,21 +22,37 @@
 from quantus.functions.normalise_func import normalise_by_average_second_moment_estimate
 from quantus.functions.perturb_func import uniform_noise, perturb_batch
 from quantus.helpers.utils import expand_attribution_channel
+from quantus.helpers.enums import ModelType, DataType, ScoreDirection, EvaluationCategory
 
 
 class RelativeRepresentationStability(BatchedPerturbationMetric):
     """
-    Relative Output Stability leverages the stability of an explanation with respect
-    to the change in the output logits
+    Relative Representation Stability leverages the stability of an explanation with respect
```

Comment:
Oh, that's a typo I did 🤦 
Thanks for noticing!

Reply:
np!